public/app/plugins/panel/timeseries/models.gen.ts

```diff
@@ -0,0 +1,14 @@
+//~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+// This file was autogenerated by cuetsy. DO NOT EDIT!
+//~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+import * as ui from '@grafana/schema';
+
+export const modelVersion = Object.freeze([0, 0]);
```

Comment:
What does `modelVersion` mean and where is it used? (sorry for newb question, curious about the context)

Reply:
Ha we are all newb's here as we figure out what structures/formats make sense.  The idea with `modelVersion` is that it will reference the [scumata](https://docs.google.com/document/d/1qp3flLb6q1dPqTmHQ6MV-o6lOmlctfMNpzTx2MdzucY/edit#heading=h.s9aa2qw61rzk) version defined for each object that manages migrations.  Currently that is each panel plugin and the dashboard schema

Reply:
Thanks @ryantxu

Reply:
Not used anywhere yet, but will be. It represents the number of the latest [scuemata version](https://docs.google.com/document/d/1qp3flLb6q1dPqTmHQ6MV-o6lOmlctfMNpzTx2MdzucY/edit#heading=h.ps08c938afna) in the underlying CUE file - lineage version and schema version, respectively.

You can see it creeping in with `panelVersion` in some of the files in e.g. #39412 , though that's not the final form it will take. Eventually, scuemata versions are intended to replace `schemaVersion` in the main dashboard object, and supplement the information on each panel, and each target (when we get to datasources).